src/datamonkey-usage.md

```diff
@@ -0,0 +1,235 @@
+```js
+import * as Plot from "npm:@observablehq/plot";
+import * as d3 from "npm:d3";
+import { weeklyPlot, sitesAndSequences } from "./components/usage.js";
+
+// A list of methods
+const methods = [
```

Comment:
not used? but mapping this to a function w a template could be nice. sounds like maybe you were going that way anyhow..